Sources/armory/logicnode/BloomGetNode.hx

```diff
@@ -7,14 +7,11 @@ class BloomGetNode extends LogicNode {
 	}
 
 	override function get(from:Int):Dynamic {
-        if(from == 0) {
-            return armory.renderpath.Postprocess.bloom_uniforms[0];
-        } else if (from == 1) {
-            return armory.renderpath.Postprocess.bloom_uniforms[1];
-        }  else if (from == 2) {
-            return armory.renderpath.Postprocess.bloom_uniforms[2];
-        } else {
-            return 0.0;
-        }
-    }
+		return switch (from) {
+			case 0: armory.renderpath.Postprocess.bloom_uniforms[0];
+			case 1: armory.renderpath.Postprocess.bloom_uniforms[1];
+			case 2: armory.renderpath.Postprocess.bloom_uniforms[2];
+			default: null;
```

Comment:
`null` and `0.0` are different values, I'm not sure which is better here. But actually, this is rather a case for an exception because this line is only triggered if the node has more outputs socket than implemented in the .hx source. But that's something for a future PR.

Reply:
I will leave these values as 0.0 for now, because i don't know what effect this will cause and i can't test now. Thanks!